src/rest_settings.ts

```diff
@@ -225,6 +225,170 @@ const paths = {
    * @method get
    */
   queryTokenHistory: '/tokens/history/query',
+
+  // Notifiaction
+
+  /*
+  * Waits for the CronjobCreated notification and returns the result
```

Comment:
S.o., auch für alle anderen Routen.

```suggestion
  * Waits for a CronjobCreated notification and resolves with the notification's content.
```